tests/resources/serializers/test_dcat_serializer.py

```diff
@@ -0,0 +1,261 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Resources serializers tests."""
+
+import pytest
+
+from invenio_rdm_records.resources.serializers import DCATSerializer
+
+
+@pytest.fixture
+def full_modified_record(full_record):
+    full_record["metadata"]["additional_descriptions"][0]["lang"]["title"] = {
```

Comment:
Additional fields do not only test DCAT schema, there are just all the fields that can be in the record, and are absent in the `full_record`. Some of them are not even serialized by DCAT, but still good to have them in test in case the serializer changes - we would know.

Reply:
I feel that hints towards a need to update `full_record` with all the fields (so that it is truly "full"). But I get that this might actually cause other tests to fail (which would be good to discover though :) ).

A silly alternative is to create a new `fullest_record` fixture or similar and open an issue to swap it later on.